web/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
-    "build": "react-scripts build",
+    "build": "SKIP_PREFLIGHT_CHECK=true react-scripts build",
```

Comment:
with latest version of create react app it may not be necessary to skip preflight check as it has an upgraded version of eslint.